daphne/tests/test_http_response.py

```diff
@@ -72,7 +72,6 @@ def test_headers(self, headers):
         headers=http_strategies.headers(),
         body=http_strategies.http_body(),
     )
-    @settings(perform_health_check=False)
```

Comment:
this was suppressing all health checks, I've just left us suppressing the 'too slow' one only